src/main/java/org/openstreetmap/atlas/tags/annotations/extraction/SpeedExtractor.java

```diff
@@ -18,33 +23,40 @@
     private static final SpeedValidator VALIDATOR = new SpeedValidator();
     private static final String SINGLE_SPACE = " ";
     private static final Logger logger = LoggerFactory.getLogger(SpeedExtractor.class);
+    private static final JsonObject IMPLICIT_SPEED_MAP = new Gson().fromJson(
+            new JsonReader(
+                    new InputStreamReader(SpeedExtractor.class.getClassLoader().getResourceAsStream(
+                            "org/openstreetmap/atlas/tags/annotations/implicit-speed-values.json"))),
```

Comment:
To avoid issues during potential future refactorings, we should not list package names like this. It would be better to do:
```
... new InputStreamReader(Tag.class.getResourceAsStream("implicit-speed-values.json")) ...
```
Here `Tag` is in the same package as the resource you are trying to fetch.